packages/@coorpacademy-components/src/atom/button-link/index.tsx

```diff
@@ -74,10 +82,28 @@ const ButtonLink = (props: ButtonLinkProps) => {
     disabled && style.disabled
   );
 
+  const [hovered, setHovered] = useState(false);
+
   const handleOnClick = useCallback(() => onClick(), [onClick]);
 
   const handleOnKeyDown = useCallback(event => onKeyDown(event), [onKeyDown]);
 
+  const handleMouseOver = useCallback(() => setHovered(true), [setHovered]);
+
+  const handleMouseLeave = useCallback(() => setHovered(false), [setHovered]);
+
+  const _customStyle = useMemo(() => {
+    return {
+      ...customStyle,
+      ...((hoverBackgroundColor || hoverColor) && hovered
+        ? {
+            backgroundColor: hoverBackgroundColor,
+            color: hoverColor
+          }
+        : null)
+    };
+  }, [hoverBackgroundColor, hoverColor, hovered, customStyle]);
+
   if (link) {
     return (
       <Link
```

Comment:
maybe, it' a good idea to enable the same logic for the Link component too ?

Reply:
Ah, the Link itself already have hover state logic: https://github.com/CoorpAcademy/components/blob/master/packages/%40coorpacademy-components/src/atom/link/index.js#L9